LICENSE

```diff
@@ -186,7 +186,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright 2015 Kaliop SAS
+   Copyright 2015-2018 Kaliop SAS
```

Comment:
Shouldn't this copyrighted to Kuzzle instead now?

Reply:
you're right, fixed.